app/views/service_providers/_form.html.slim

```diff
@@ -7,7 +7,8 @@
   as: :collection_select,
   collection: current_user.scoped_user_groups,
   include_blank: '',
-  selected: selected_user_group
+  selected: selected_user_group,
+  disabled: !current_user.user_group?
```

Comment:
per my naming recommendation above, I also think this would be clearer if it read something like `disabled: !can_edit_user_group?`